### PR TITLE
Add seed data for a simple hackney booking request

### PR DESCRIPTION
**Pull request by AndrewVos**
Also adds the first user for gds-sso.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+user = User.find_or_initialize_by(
+  name: "Andrew",
+)
+user.organisation_content_id = "ac7112c3-e3cf-45cd-a8ff-9ba827b8e7ef"
+user.save!
+
+FactoryGirl.create(
+  :hackney_booking_request,
+  location_id: "183080c6-642b-4b8f-96fd-891f5cd9f9c7",
```
Comment (benlovell) [inline db/seeds.rb]: You may as well remove these two attributes and just create the default booking request at the Hackney location.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,11 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+user = User.find_or_initialize_by(
```
Comment (benlovell) [inline db/seeds.rb]: May as well `find_or_create_by` and move the `organisation_content_id` in to its parameters.

File: db/seeds.rb (hunk 1 above)
Comment (AndrewVos) [inline db/seeds.rb]: Cool, also added the `booking_manager` permission to the user.

**Comment by AndrewVos** [comment]
@benlovell updated ^

**Pull request merged**